src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepExecution.java

```diff
@@ -49,7 +47,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class BuildTriggerStepExecution extends AbstractStepExecutionImpl {
+public class BuildTriggerStepExecution extends StepExecution {
```

Comment:
As per constructor Javadoc, revert:

```suggestion
public class BuildTriggerStepExecution extends AbstractStepExecutionImpl {
```

Reply:
Does the trick. Thank you!